docs/WatchPlugins.md

```diff
@@ -157,8 +157,6 @@ For stability and safety reasons, only part of the global configuration keys can
 - [`bail`](configuration.html#bail-number--boolean)
 - [`changedSince`](cli.html#--changedsince)
 - [`collectCoverage`](configuration.html#collectcoverage-boolean)
-- [`collectCoverageFrom`](configuration.html#collectcoveragefrom-array)
```

Comment:
I think watch plugins should still be able to set these, somehow... Might have to solve watchplugins setting project options before we can land this